packages/components/src/core/Panel/components/PanelHeaderClose/index.tsx

```diff
@@ -0,0 +1,28 @@
+import Button from "src/core/Button";
+import { StyledPanelHeaderClose } from "./style";
+
+export interface PanelHeaderCloseProps {
+  onClick?: React.MouseEventHandler<HTMLDivElement>;
+  closeButtonComponent?: React.ReactNode;
```

Comment:
Maybe PascalCase for the component prop?

```suggestion
  CloseButtonComponent?: React.ReactNode;
```